x-pack/plugins/security_solution/public/management/components/artifact_list_page/artifact_list_page.test.tsx

```diff
@@ -0,0 +1,919 @@
+/*
```

Comment:
Suggestion: This is a very good test file but seems a bit long, could we split this in different files for each area?

Reply:
yeah, was thinking the same thing once i was done with the test. I will open an issue and will do it in a subsequent PR.